pkg/cosign/verify.go

```diff
@@ -585,6 +587,10 @@ func VerifyImageSignatures(ctx context.Context, signedImgRef name.Reference, co
 		}
 	}
 
+	if co.NewBundleFormat {
+		return nil, false, errors.New("bundle support for image signatures is not yet implemented")
```

Comment:
Interesting! I didn't know this feature existed. For policy controller users they will definitely be using image attestations, but do we have an idea about how common are / what use cases people are using image signatures?

Reply:
Good question! IIRC, a Rekor analysis showed pretty strong usage of signatures, but I expect attestations will outpace them. I'd like to fast follow with support for image signatures. Assuming we go with option three in https://github.com/sigstore/cosign/issues/3927, it will share much of the same code as image attestations.